tests/topotests/conftest.py

```diff
@@ -297,6 +297,59 @@ def check_for_memleaks():
         pytest.fail("memleaks found for daemons: " + " ".join(daemons))
 
 
+def check_for_core_dumps():
+    dumps = []
+    tgen = get_topogen()  # pylint: disable=redefined-outer-name
+    latest = []
+    existing = []
+
+    if tgen is not None:
+        logdir = tgen.logdir
+        if hasattr(tgen, "cores_existing_files"):
+            existing = tgen.cores_existing_files
+        cores = glob.glob(os.path.join(logdir, "*/*.dmp"))
+
+    if tgen is not None:
+        tgen.cores_existing_files = existing
```

Comment:
I think your intention here is to track and discount cores from earlier tests, but you're not adding new files from `cores` to `existing` and not discounting files in `existing` from `cores` so this is a NOOP currently.

Reply:
yep, but I was following the code from the other functions and left it in because of that.  They are all no-ops for this particular pattern.  I just don't know enough about the system to understand what the goal was here.  Do you?